SECURITY.md

```diff
@@ -0,0 +1,46 @@
+# Security Notice
+
+## What?
+
+This is the security notice for all GDS (Government Digital Service) `alphagov` repositories. The notice explains how vulnerabilities should be reported to GDS. At GDS there are cyber security and information assurance teams, as well as security-conscious people within the programmes, that assess and triage all reported vulnerabilities.
+
+## Reporting a Vulnerability
+
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, great! We would like to know so we can fix it.
+
+If you believe a vulnerability affects the data (release of sensitive data or personal identifiable information) of a system, send details to [disclosure@digital.cabinet-office.gov.uk], include:
+- the website, page or repository where the vulnerability can be observed
+- a brief description of the type of vulnerability and any related [OWASP category]
```

Comment:
Isn't the bare minimum we require is a description? 
```suggestion
- a brief description of vulnerability
```
I feel like asking the person to try and classify the vulnerability is increases the barrier to them actually reporting it.

Reply:
I've separated the two points and said the type is optional - I think this is useful information in what the researcher/bug finder believes the vulnerability is.